ajax/libs/colors/package.json

```diff
@@ -19,5 +22,14 @@
     "name": "@mrmrs",
     "email": "hi@mrmrs.cc",
     "url": "http://mrmrs.cc"
-  }
+  },
+  "npmName": "colors.css",
```

Comment:
@samme Adding npm auto-update part should in `Add colors@3.0.0 with npm auto-update` part, not in `Enrich and update colors package.json` part.